man/man8/facter.8

```diff
@@ -35,6 +35,7 @@ If no queries are given, then all facts will be returned\.
       \fB\-\-external-dir\fR arg           A directory to use for external facts\.
       \fB\-\-help\fR                       Print help and usage information\.
 \fB\-j, [ \-\-json ]\fR                     Output facts in JSON format\.
+      \fB\-\-show-legacy\fR                Add legacy fact entries to output\.
```

Comment:
Should this match the --help output?  i.e. `Show legacy facts when querying all facts.`